lib/Doctrine/ORM/Configuration.php

```diff
@@ -441,6 +441,7 @@ public function addCustomStringFunction($name, $className)
      * @param string $name
      *
      * @return string|null
+     * @psalm-return ?class-string
```

Comment:
When I see https://github.com/doctrine/data-fixtures/pull/332, I'm not sure we should use `psalm-return`… I think we could just use return and remove line 443… WDYT? Apparently it's recognized by recent versions of phpstan as well: https://github.com/doctrine/data-fixtures/issues/337

Reply:
Yeah, Phpstan reads most psalm annotations and understand almost everything psalm uses. I kept the return annotation mainly for other tools like PHPStorm who don't understand it yet. For example, when I added types in https://github.com/Roave/BetterReflection, Ocramius prefered to keep standard notations clean as long as psalm's and phpstan's 
notations are not understood widely: https://github.com/Roave/BetterReflection/pull/531#pullrequestreview-342848231

For `class-string`, it may seem a trivial difference in PHPStorm (string vs mixed), but when we will replace things like `RandomClass[]` by their more detailed `list<RandomClass>`, PHPStorm won't even read it as an array, and autocompletion will be broken. Depending on the IDEs used by maintainers, it can be a big drawback